vic/drivers/classic/src/get_global_param.c

```diff
@@ -1289,46 +1297,46 @@ get_global_param(FILE *gp)
                 "the vegparam file, you MUST make sure the veg param file "
-                "contains 1 line of 12 monthly vegcover values for EACH veg "
+                "contains 1 line of 12 monthly fcanopy values for EACH veg "
                 "tile in EACH grid cell, and you MUST specify "
-                "\"VEGPARAM_VEGCOVER\" as \"TRUE\" in the global parameter "
-                "file.  Alternatively, if you want VIC to read vegcover values "
-                "from the veg library file, set \"VEGCOVER_SRC\" to "
+                "\"VEGPARAM_FCAN\" as \"TRUE\" in the global parameter "
+                "file.  Alternatively, if you want VIC to read fcanopy values "
+                "from the veg library file, set \"FCAN_SRC\" to "
                 "\"FROM_VEGLIB\" in the global parameter file.  In "
-                "either case, the setting of \"VEGPARAM_VEGCOVER\" must be "
+                "either case, the setting of \"VEGPARAM_FCAN\" must be "
                 "consistent with the contents of the veg param file (i.e. "
-                "whether or not it contains vegcover values).");
+                "whether or not it contains fcanopy values).");
     }
-    if (options.VEGCOVER_SRC == FROM_VEGLIB && !options.VEGLIB_VEGCOVER) {
-        log_err("\"VEGCOVER_SRC\" was specified as \"FROM_VEGLIB\", but "
-                "\"VEGLIB_VEGCOVER\" was set to \"FALSE\" in the global "
-                "parameter file.  If you want VIC to read vegcover values from "
+    if (options.FCAN_SRC == FROM_VEGLIB && !options.VEGLIB_FCAN) {
+        log_err("\"FCAN_SRC\" was specified as \"FROM_VEGLIB\", but "
+                "\"VEGLIB_FCAN\" was set to \"FALSE\" in the global "
+                "parameter file.  If you want VIC to read fcanopy values from "
                 "the veglib file, you MUST make sure the veg lib file "
-                "contains 1 line of 12 monthly vegcover values for EACH veg "
+                "contains 1 line of 12 monthly fcanopy values for EACH veg "
                 "class, and you MUST specify "
-                "\"VEGLIB_VEGCOVER\" as \"TRUE\" in the global parameter "
-                "file.  Alternatively, if you want VIC to read vegcover values "
-                "from the veg param file, set \"VEGCOVER_SRC\" to "
+                "\"VEGLIB_FCAN\" as \"TRUE\" in the global parameter "
+                "file.  Alternatively, if you want VIC to read fcanopy values "
+                "from the veg param file, set \"FCAN_SRC\" to "
                 "\"FROM_VEGPARAM\" in the global parameter file.  In "
-                "either case, the setting of \"VEGLIB_VEGCOVER\" must be "
+                "either case, the setting of \"VEGLIB_FCAN\" must be "
                 "consistent with the contents of the veg lib file (i.e. "
-                "whether or not it contains vegcover values).");
+                "whether or not it contains fcanopy values).");
```

Comment:
So, actually, not only did I not add such an error, but I removed the old error for using "GLOBAL_LAI" because I figured that option is old enough by now that we can stop maintaining that error.  So, I guess my question now is: is there a limit to how old deprecated options can be before we stop checking for them and raising errors?  Should I reinstate the check on GLOBAL_LAI?